src/helpers/environment.ts

```diff
@@ -5,5 +5,6 @@ export default {
   HOME: process.env.HOME || process.env.ENV || null,
   WORKPATH: process.env.WORKPATH || '/home/myproxy',
   isProduction: (): boolean =>
-    (process.env.NODE_ENV || process.env.ENV) === 'production'
+    (process.env.NODE_ENV || process.env.ENV) === 'production',
+  isTest: (): boolean => (process.env.NODE_ENV || process.env.ENV) === 'test'
```

Comment:
this means that during development, we have to run this with test now? I don't think devs are ready for that yet. Could we use production flag?